net/mtxclient/Portfile

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+PortGroup           cxx11 1.1
+PortGroup           compiler_blacklist_versions 1.0
+#NOTE: requires C++14
+
+github.setup        mujx mtxclient 2f519d2
+version             2018-07-14
+categories          net chat devel
+platforms           darwin
+license             GPL-3
+maintainers         {@scarface-one disroot.org:scarface} openmaintainer
+description         Client API for Matrix
+long_description    Client API library for the Matrix protocol, built on top of Boost.Asio.
+checksums           rmd160  28330b8f5c028792aaa5a1dbdd3738aea4854e7d \
+                    sha256  760a03fabae4cca250a5a7aa1328e7fb922186b431c1b65530ad083dcb9fda4c \
+                    size    92272
+
+#C++14 was first fully supported by clang 3.4 aka apple clang 503.0.38
+compiler.blacklist-append {clang < 503.0.38}
+
+#The patch is necessary since cmake has a hard time finding libsodium provided by macports
+patchfiles          patch-CMakeList.txt.diff
+
+configure.args-append   -H. \
+                        -Bbuild \
+                        -DBUILD_LIB_TESTS=OFF \
+                        -DBUILD_LIB_EXAMPLES=OFF \
+                        -DOPENSSL_ROOT_DIR=/usr/local/opt/openssl \
```

Comment:
`/usr/local/opt` can't be right in a macports context.

Reply:
I had copied it from the build instructions. Good catch. It seems that regardless of what the value of the variable is, as long as it is defined, the build succeeds. I've fixed it to point to the version installed by macports though